From 2330f3b82cbc39cc9c6c4fd82db15cfa7eb6c0c3 Mon Sep 17 00:00:00 2001
From: Jared Morrow <jm@joyent.com>
Date: Mon, 24 Jul 2017 12:30:11 -0600
Subject: [PATCH] MANTA-3109 manta log uploader retries multiple times in
 parallel Reviewed by: Joshua M. Clulow <jmc@joyent.com> Approved by: Joshua
 M. Clulow <jmc@joyent.com>

---
 deps/manta-scripts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/manta-scripts b/deps/manta-scripts
index cd72e80..4b86dd1 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit cd72e801576de5a4b1aec35a7f79409d3f1eebb5
+Subproject commit 4b86dd132d68cb177f066a83d90b8fdc9ac2cde9
-- 
2.21.0

